src/ApiClient.php

```diff
@@ -7,7 +7,6 @@
 use League\Container\Argument\LiteralArgument;
 use League\Container\Container;
 use OpenEuropa\CdtClient\Contract\ApiClientInterface;
-use OpenEuropa\CdtClient\Contract\EndpointInterface;
```

Comment:
I removed this interface to simplify these endpoint classes. Since endpoints are only called by this class, the `execute` method overridden (including its types) by each endpoint was not a very good idea. Please let me know what you think.